doc/3/core-classes/kuzzle/connect/index.md

```diff
@@ -0,0 +1,26 @@
+---
+code: true
+type: page
+title: Connect
+description: Connects the SDK to Kuzzle
+---
+
+# Connect
+
+Connects to Kuzzle using the subsequent protocol `connect` method. 
+
+Subsequent call have no effect if the SDK is already connected.
+
+## Arguments
+
+```java
+public void connect() throws Exception;
+```
+
+## Throws
+
+Throws an `Exception`.
```

Comment:
This is because it's an interface so it depends how you implement connect (with which library etc) and it could throw any exception, even custom exceptions which are part of a library